PythonPractice/TicTacToe/tictactoe.py

```diff
@@ -0,0 +1,155 @@
+class SpaceTakenError(Exception):
+    pass
+
+class BoardTooSmallError(Exception):
+    pass
+
+class TicTacToe:
```

Comment:
If you're programming in Python 2.x, this should be `class TicTacToe(object):`; this makes it a "new-style" rather than "old-style" class.  The distinction is subtle - as, really, are many of the distinctions between Python 2.x and 3.x - and in both cases they more or less boil down to old-and-busted vs. new-hotness.